src/java.desktop/share/classes/sun/swing/FilePane.java

```diff
@@ -1197,48 +1196,53 @@ public Component getTableCellRendererComponent(JTable table, Object value,
 
             } else if (value instanceof Long len) {
                 /*
-                 * Code block is relevant to linux
-                 * File size display upto 1 decimal precision
+                 * Code block is relevant to Linux.
+                 * File size is display up to 1 decimal precision.
```

Comment:
```suggestion
                 * File size is displayed up to 1 decimal precision.
```
Sorry, I missed it yesterday when I added _“is”_.